pkg/minikube/constants/constants.go

```diff
@@ -29,9 +29,10 @@ const (
 	MiniShiftEnvPrefix               = "MINISHIFT"      // Prefix for the environmental variables
 	MiniShiftHomeEnv                 = "MINISHIFT_HOME" // Environment variable used to change the Minishift home directory
 	VersionPrefix                    = "v"
-	MinimumSupportedOpenShiftVersion = "v1.4.1"
-	MinimumOcBinaryVersion           = "v3.6.1" // For provisioning OpenShift versions < v3.7.0 , we need to use oc v3.6.1. Refer openshift/origin#17821
-	BackwardIncompatibleOcVersion    = "v3.7.0" // From v3.7.0 oc binary will not provision older versions of OpenShift instance. Refer #1417
+	MinimumSupportedOpenShiftVersion = "v3.6.1"
+	MinimumOcBinaryVersion           = "v3.6.1"  // For provisioning OpenShift versions < v3.7.0 , we need to use oc v3.6.1. Refer openshift/origin#17821
+	BackwardIncompatibleOcVersion    = "v3.7.0"  // From v3.7.0 oc binary will not provision older versions of OpenShift instance. Refer #1417
+	RefactoredOcVersion              = "v3.10.0" // From v3.10.0 oc binary is refactored and don't have lot of options by default.
```

Comment:
ATM has no real meaning in code, right?
So what will this do or mean in the future ... ?